en/orm/retrieving-data-and-resultsets.rst

```diff
@@ -1039,10 +1014,6 @@ Some other examples of the collection methods being used with result sets are::
     });
 
     // Create an associative array from result properties
-    // Prior to 3.6.0
-    $articles = TableRegistry::get('Articles');
-
-    $articles = TableRegistry::getTableLocator()->get('Articles');
     $results = $articles->find()->contain(['Authors'])->all();
```

Comment:
But shouldnt those then be class props?

    $this->Articles->find()

etc
As you probably loaded them via trait.

Reply:
Also:
We either expect them to know how to load those tables, or ideally add the call:

    $this->loadModel('Articles');

Especially, if the current controller is not necessarily expected to have this as default model.